src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.util.Colors;
+
+public class Circle extends Figure {
+
+    private int rad;
+
+    public Circle(Colors color) {
+        super(color);
+        this.rad = getRandomNum();
+    }
+
+    public Circle(Colors color, int rad) {
+        super(color);
+        this.rad = rad;
+    }
```

Comment:
And this constructor (a little diffrent for each subclass of course - depending on the fields particular subclass has) should be used in all subclasses.

Reply:
i thing u would like it